webview-ui/src/context/ExtensionStateContext.tsx

```
@@ -20,6 +20,8 @@ interface ExtensionStateContextType extends ExtensionState {
 	mcpServers: McpServer[]
 	mcpMarketplaceCatalog: McpMarketplaceCatalog
 	filePaths: string[]
+	totalTasksSize: number | null
```

lets use `totalTasksSize?: number` instead, since we don't use the `| null` pattern anywhere else

---

I would point out, null and undefined aren't exactly the same thing, but probably OK to use in this case.

---

@dcbartlett @saoudrizwan Talked with Saoud, will keep this way. The reason being that if the operation fails we want it to silently fail and just return null. In this way we can just not display the size.